finality-aleph/src/metrics.rs

```diff
@@ -21,52 +23,56 @@ use crate::aleph_primitives::BlockHash;
 const MAX_BLOCKS_PER_CHECKPOINT: usize = 5000;
 
 const LOG_TARGET: &str = "aleph-metrics";
+const HISTOGRAM_BUCKETS: [f64; 11] = [
+    1., 5., 25., 100., 150., 250., 500., 1000., 2000., 5000., 10000.,
```

Comment:
Even then I would rather configure only parameters for `exponential_buckets` (or another appropriate function) rather than all buckets by hand. Plus changing this to be configurable this way in this PR would be quite annoying.

Reply:
ofc, just sharing my exp from past work on issues like that.

Reply:
I can agree with both. If we'd like to validate statements like "import takes at most `X` ms in 95% cases", then we have to have an `X` bucket. 
Since we'll probably iterate on exact values of `X` for different checkpoints, I'm going to create a config with separate buckets definition for each histogram metrics, which will contain our currently agreed Xes in the middle and other buckets exponentially lower and higher.

Reply:
but yeah, the config part is probably out of scope of this pr, so I will make only some preparations for it